libs/core/include/core/serializers/base_types.hpp

```diff
@@ -839,5 +841,64 @@ struct ForwardSerializer<fixed_point::FixedPoint<I, F>, D>
   }
 };
 
+template <typename D>
+struct ForwardSerializer<fixed_point::FixedPoint<64, 64>, D>
+{
+public:
+  using Type       = fixed_point::FixedPoint<64, 64>;
+  using DriverType = D;
+
+  template <typename Interface>
+  static void Serialize(Interface &interface, Type const &n)
+  {
+    interface << static_cast<uint64_t>(n.Data()) << static_cast<uint64_t>(n.Data() >> 64);
```

Comment:
can we put parentheses around "static_cast<uint64_t>(n.Data() >> 64" so that the order of operations is crystal clear? it took a few seconds to figure out what was happening here

Reply:
fixed